internal/controller/eventing/integrationtests/without_apirule_crd/integration_test.go

```diff
@@ -0,0 +1,103 @@
+package controller_switching
+
+import (
+	"os"
+	"testing"
+
+	eventingv1alpha1 "github.com/kyma-project/eventing-manager/api/v1alpha1"
+	"github.com/kyma-project/eventing-manager/test/matchers"
+	"github.com/kyma-project/eventing-manager/test/utils"
+	testutils "github.com/kyma-project/eventing-manager/test/utils/integration"
+	"github.com/onsi/gomega"
+	gomegatypes "github.com/onsi/gomega/types"
+)
+
+const (
+	projectRootDir = "../../../../../"
+)
+
+var testEnvironment *testutils.TestEnvironment //nolint:gochecknoglobals // used in tests
+
+// TestMain pre-hook and post-hook to run before and after all tests.
+func TestMain(m *testing.M) {
+	// Note: The setup will provision a single K8s env and
+	// all the tests need to create and use a separate namespace
+
+	// setup env test
+	var err error
+	testEnvironment, err = testutils.NewTestEnvironment(testutils.TestEnvironmentConfig{
+		ProjectRootDir:            projectRootDir,
+		CELValidationEnabled:      true,
+		APIRuleCRDEnabled:         false,
+		ApplicationRuleCRDEnabled: true,
+		NATSCRDEnabled:            true,
+		AllowedEventingCR:         nil,
+	})
+	if err != nil {
+		panic(err)
```

Comment:
Please don't use panics. Security scanners will not be happy about it.

Reply:
I will create a follow up ticket for it because we need to update it in all integration tests